substrate/utils/frame/omni-bencher/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "polkadot-omni-bencher"
```

Comment:
Can we please call it just `omni-bencher` or `frame-omni-bencher`?

Reply:
Yes i also though about this, but it exposes exactly the host functions of the Polkadot Host.  
So if we cann it `frame-*` then someone could assume that it works for all FRAME runtimes?  
Anyway i agree maybe that is still better than to put polkadot directly on the name.

Reply:
wait, this doesn’t work with all FRAME runtimes?

Reply:
Solochains could have custom host functions and they would not work with this benchmarker. For Polkadot & Parachains this is not really relevant.

Reply:
It would be nice of this was made easily(ish) customisable for at least Polkadot/Kusama parachains.

Reply:
Renamed to `frame-omni-bencher` now.